frontend/components/common/Table.tsx

```diff
@@ -332,21 +363,25 @@ const Table = ({
           </select>
           <button
             style={{ marginRight: '0.5rem' }}
-            onClick={() => gotoPage(pageCount - 1)}
+            className="is-light is-small"
+            onClick={() => nextPage()}
             disabled={!canNextPage}
           >
-            <Icon name="chevron-right" />
+            {'>'}
           </button>
           <button
             style={{ marginRight: '0.5rem' }}
-            onClick={() => nextPage()}
+            className="is-light is-small"
+            onClick={() => gotoPage(pageCount - 1)}
```

Comment:
Can `pageCount` ever be 0?

Reply:
I don't think. CanPrevious is automatically set to disabled when we are at page 1, so I don't see how that might happen.